sources/packages/api/src-sql/User/Drop-User.sql

```diff
@@ -0,0 +1,6 @@
+-- DROP Indicies
+DROP INDEX IF EXISTS "idx_users_id_20210115_5_00";
```

Comment:
There are 3 indexes in the user table but only 2 are being removed here. Can you check it, please?